package.json

```diff
@@ -26,6 +26,9 @@
     "start": "ember serve",
     "test": "ember test"
   },
+  "overrides": {
+    "ember-cli-babel": "^7.26.11"
```

Comment:
added this because the console warnings about a bunch of different ember-cli-babel versions kinda make this app look unmaitained 😅